src/Storage/StreamWrapper.php

```diff
@@ -250,8 +250,13 @@ public function stream_close()
     // @codingStandardsIgnoreStart
     /**
      * Callback handler for trying to seek to a certain location in the stream.
+     *
+     * @param int $ofset The stream offset to seek to
```

Comment:
`offset` :)

Also, below - `$whence` appears to be optional. Do you think we should default its value?